src/pkg/services/m365/m365.go

```diff
@@ -47,6 +47,48 @@ func UsersCompat(ctx context.Context, acct account.Account) ([]*User, error) {
 	return users, errs.Failure()
 }
 
+// UsersCompatNoInfo returns a list of users in the specified M365 tenant.
+// TODO: Remove this once `Info` is removed from the `User` struct and callers
+// have switched over
+func UsersCompatNoInfo(ctx context.Context, acct account.Account) ([]*User, error) {
+	errs := fault.New(true)
+
+	users, err := usersNoInfo(ctx, acct, errs)
+	if err != nil {
+		return nil, err
+	}
+
+	return users, errs.Failure()
+}
+
+// usersNoInfo returns a list of users in the specified M365 tenant - with no info
+// TODO: Remove this once we remove `Info` from `Users` and instead rely on the `GetUserInfo` API
+// to get user information
+func usersNoInfo(ctx context.Context, acct account.Account, errs *fault.Bus) ([]*User, error) {
+	uapi, err := makeUserAPI(acct)
+	if err != nil {
+		return nil, clues.Wrap(err, "getting users").WithClues(ctx)
+	}
+
+	users, err := discovery.Users(ctx, uapi, errs)
+	if err != nil {
+		return nil, err
+	}
+
+	ret := make([]*User, 0, len(users))
```

Comment:
Info values will be empty in the `User` struct. But that's understood from function name itself.

Reply:
To be safe - I'm going to clone the `User` struct also

Reply:
Fixed